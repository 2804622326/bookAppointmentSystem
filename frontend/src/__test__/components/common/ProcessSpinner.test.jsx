import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import ProcessSpinner from '../../../components/common/ProcessSpinner.jsx';

test('renders spinner with message snapshot', () => {
  const { asFragment } = render(
    <ProcessSpinner message="Loading" animation="border" />
  );
  expect(asFragment()).toMatchSnapshot();
});
