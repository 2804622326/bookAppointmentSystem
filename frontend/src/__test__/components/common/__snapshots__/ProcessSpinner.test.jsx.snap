// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders spinner with message snapshot 1`] = `
<DocumentFragment>
  <div
    class="text-center"
  >
    <span
      aria-hidden="true"
      class="spinner-border spinner-border-sm"
      role="status"
    />
    <span
      class="sr-only"
    >
      Loading
    </span>
  </div>
</DocumentFragment>
`;
